Formula/pldiff.rb

```diff
@@ -0,0 +1,16 @@
+class Pldiff < Formula
+  desc "Compares plist files line by line"
+  homepage "https://github.com/scottrigby/pldiff"
+  url "https://github.com/scottrigby/pldiff/archive/1.0.4.tar.gz"
+  sha256 "0093027aee4ceb049902a9fcea28923477c87c2f88813e2370fb4c12195b9ccd"
+
+  depends_on "colordiff" => :recommended
+
+  def install
+    system "/bin/sh", "install.sh", prefix
+  end
+
+  test do
+    system "false"
```

Comment:
@woodruffw just following up… I added the test https://github.com/scottrigby/homebrew-macos-configurator/commit/ebda1ab2f82213d63b8cc30c24f82985806bec18, and it works perfectly:
```diff
diff --git a/Formula/pldiff.rb b/Formula/pldiff.rb
index ca9c348..80057a5 100644
--- a/Formula/pldiff.rb
+++ b/Formula/pldiff.rb
@@ -11,6 +11,10 @@ class Pldiff < Formula
   end
 
   test do
-    system "false"
+    system "defaults write #{testpath}/a foo -string bar"
+    system "defaults write #{testpath}/b foo -string baz"
+    o = shell_output("echo $(pldiff #{testpath}/a.plist #{testpath}/b.plist)")
+    assert_match "- <string>bar</string>", o
+    assert_match "+ <string>baz</string>", o
   end
 end
```
I think I'm all sorted for this formula. Thanks again!